frontend_tests/casper_tests/05-subscriptions.js

```diff
@@ -28,20 +28,35 @@ casper.waitForSelector('.sub_unsub_button.checked', function () {
 
 casper.then(function () {
     casper.test.assertExists('#user-checkboxes [data-name="cordelia@zulip.com"]', 'Original user list contains Cordelia');
-    casper.test.assertExists('#user-checkboxes [data-name="hamlet@zulip.com"]', 'Original user list contains King Hamlet');
+    casper.test.assertExists('#user-checkboxes [data-name="othello@zulip.com"]', 'Original user list contains Othello');
 });
+
+casper.waitForSelector("#copy-from-stream-expand-collapse", function () {
+    casper.click('#copy-from-stream-expand-collapse');
+    casper.test.assertExists('#stream-checkboxes [data-name="Scotland"]', 'Original stream list contains Scotland');
```

Comment:
As the click is changing the style from `display: none` to `display: block`, I now moved the asserts into a `casper.waitUntilVisible`.